Assets/RGLUnityPlugin/Scripts/RadarModels/RadarConfiguration.cs

```diff
@@ -48,16 +48,31 @@ public class RadarConfiguration
         [Tooltip("Maximum elevation angle (up)")]
         [Range(-180.0f, 180.0f)] public float maxElevationAngle;
 
+        [Tooltip("The frequency of the radar (in GHz)")]
+        [Range(0.0f, 100.0f)] public float frequency = 79.0f;
+
+        [Tooltip("The power transmitted by the radar (in dBm)")]
+        [Range(0.0f, 100.0f)] public float powerTransmitted = 31.0f;
+
+        [Tooltip("The gain of the radar's antenna (in dBi)")]
+        [Range(0.0f, 100.0f)] public float antennaGain = 27.0f;
+
+        [Tooltip("The mean of the received noise (in dBm)")]
+        public float receivedNoiseMean = 0.0f;
```

Comment:
Value here would definitely be higher - we should check rosbags again, but as far as I remember, received noise was about 60 dBm what is quite huge.